src/lib/CollectStore.js

```diff
@@ -167,7 +176,19 @@ export default class CollectStore {
    * @returns The run result or a resulting error
    */
   runAccount (connector, account) {
+    const connection = {}
     return konnectors.run(cozy.client, connector, account)
+    .catch(error => {
```

Comment:
I think we can simplify this part, as connection is not returned at the end, we do not need it, so I think we can do:
```javascript
.then(() => this.updateKonnectorError(connector))
.catch(error => {
  this.updateKonnectorError(connector, error)
  throw error // or return Promise.reject(error), I personnally prefer to throw but there is pretty much no difference, see https://stackoverflow.com/questions/33445415/javascript-promises-reject-vs-throw
})
```
It will behave the same way (I think)